### PR TITLE
Change the way to display emojis in help command

### PR DESCRIPTION
**Pull request by xixi52**
**Change the way to display emojis in help command (category)**

**Status:**

- [x] Code changes have been tested
- [x] Code changes work without errors

**Content of the pull request:**  

- [x] This pull request changes the bot
  - [ ] This pull request includes breaking changes for the bot
  - [ ] This pull request includes new command(s) for the bot

- [ ] This pull request changes the dashboard

- [ ] This pull request **only** includes non-code changes, like changes to templates, README.md, languages, etc.

### CODE REVIEW

File: commands/General/help.js [hunk 1]
```diff
@@ -67,19 +66,19 @@ class Help extends Command {
             }
         });
 
+        let emojis = require("../../config").emojis;
```
Comment (Androz2091) [inline commands/General/help.js]: It's better to use `this.client.config.emojis`

File: config.sample.js [hunk 2]
```diff
@@ -138,6 +138,26 @@ module.exports = {
 			// RE: https://i.goopics.net/WRPWl.png
 			online: "XXXXXXXXXXX"
 			// RE: https://i.goopics.net/QdGW0.png
+		},
+		category: {
```
Comment (Androz2091) [inline config.sample.js]: I think it's better to call the object `categories` instead of `category` no?

File: commands/General/help.js [hunk 3]
```diff
@@ -67,19 +66,19 @@ class Help extends Command {
...
-            let emoji = emojis.find((e) => e.name === cat.toLowerCase()+"_category_atlanta");
             let tCommands = commands.filter((cmd) => cmd.help.category === cat);
-            embed.addField((emoji ? emoji.toString() : "")+" "+cat+" - ("+tCommands.size+")", tCommands.map((cmd) => "`"+cmd.help.name+"`").join(", "));
+            embed.addField(emojis.category[cat.toLowerCase()]+" "+cat+" - ("+tCommands.size+")", tCommands.map((cmd) => "`"+cmd.help.name+"`").join(", "));
         });
         if(message.guild){
             if(data.guild.customCommands.length > 0){
-                let emoji = emojis.find((e) => e.name === "custom_category_atlanta");
-                embed.addField((emoji ? emoji.toString() : "")+" "+message.guild.name+" | "+message.language.get("UTILS").CUSTOM_COMMANDS+" - ("+data.guild.customCommands.length+")", data.guild.customCommands.map((cmd) => "`"+cmd.name+"`").join(", "));
+                embed.addField(emojis.category["custom"]+" "+message.guild.name+" | "+message.language.get("UTILS").CUSTOM_COMMANDS+" - ("+data.guild.customCommands.length+")", data.guild.customCommands.map((cmd) => "`"+cmd.name+"`").join(", "));
```
Comment (Androz2091) [inline commands/General/help.js]: It's better to use `category.custom` instead of `category["custom"]`

File: config.sample.js (hunk 2 above)
Comment (xixi52) [reply config.sample.js]: yes ;) !

**Pull request merged**